dbtmetabase/models/metabase.py

```diff
@@ -0,0 +1,43 @@
+from dataclasses import dataclass, field
+from typing import Sequence, Optional, MutableMapping
+
+# Allowed metabase.* fields
+# Should be covered by attributes in the MetabaseColumn class
+METABASE_META_FIELDS = ["special_type", "semantic_type", "visibility_type"]
+
+
+class NonEmptyString:
```

Comment:
Maybe this was overkill on my part, but I had just finished reading some "advanced" Python books and wanted to put some concepts into practice 😄 . It's a descriptor (https://realpython.com/python-descriptors/) to be used in class attributes that automatically convert empty strings to None.